monkey/monkey_island/cc/services/attack/technique_reports/T1156.py

```diff
@@ -25,14 +24,23 @@ class T1156(AttackTechnique):
     def get_report_data():
         data = {'title': T1156.technique_title(), 'info': []}
 
-        bash_modification_info = list(mongo.db.telemetry.aggregate(T1156.query))
+        shell_startup_files_modification_info = list(mongo.db.telemetry.aggregate(T1156.query))
+
+        bash_startup_modification_info = []
+        for shell_startup_file_result in shell_startup_files_modification_info[0]['result']:
+            # only want bash startup files
+            if any(file_name in shell_startup_file_result[0] for file_name in [".bash", ".profile"]):
+                bash_startup_modification_info.append({
+                    'machine': shell_startup_files_modification_info[0]['machine'],
+                    'result': shell_startup_file_result
+                    })
```

Comment:
This should be in a method, called something like "extract_bash_modification_info", to hide the complexity.